src/main/java/org/jabref/model/entry/field/Field.java

```diff
@@ -16,8 +16,12 @@ public interface Field {
     /**
      * @return A version of the field name more suitable for display
      */
+//    default String getDisplayName() {
+//        return StringUtil.capitalizeFirst(getName());
+//    }
+
     default String getDisplayName() {
-        return StringUtil.capitalizeFirst(getName());
+        return getName();
     }
```

Comment:
Although this fixes the issue, it may also have unforseen consequences.
Better question is, where is this method called. Why is this method called instead of the method getName? Please provide some reasoning.

Reply:
Maybe, the solution is as simple as calling getName in the customize entry view model code